compiler-plugin/src/main/java/io/ballerina/stdlib/persist/compiler/DiagnosticsCodes.java

```diff
@@ -51,7 +51,14 @@ public enum DiagnosticsCodes {
     PERSIST_112("PERSIST_112", "mysql db only allow increment value by one in auto generated field",
             WARNING),
     PERSIST_113("PERSIST_113", "duplicate table name: the table name is already used in another entity",
-            ERROR);
+            ERROR),
+    PERSIST_114("PERSIST_114", "unsupported feature: M:M relationship", ERROR),
```

Comment:
Shall we say, 'unsupported features: many-to-many association is not supported yet'